javaparser-core/src/main/javacc/java.jj

```diff
@@ -711,36 +711,46 @@ StubUnit StubUnit():
     ModifierHolder modifier;
     TypeDeclaration tn = null;
     ModuleDeclaration module = null;
+    NodeList<AnnotationExpr> annotations = new NodeList<AnnotationExpr>();
```

Comment:
It would be helpful to write a brief comment saying what each of these two variables represents.